homeassistant/components/swiss_public_transport/sensor.py

```diff
@@ -86,12 +122,12 @@ def __init__(self, opendata, start, destination, name):
         self._name = name
         self._from = start
         self._to = destination
-        self._remaining_time = ""
+        self._remaining_time = None
```

Comment:
Why did this change btw

Reply:
Because should be typed an `int` and `""` is misused as `None` here.

Here it is checked if it is set and smaller 0.
https://github.com/miaucl/ha/blob/994296031f00c785d1a7999f713d5eac5ee42952/homeassistant/components/swiss_public_transport/sensor.py#L168

Reply:
This edge case was not present with the old yaml config, but with the config_flow, it seems to have a different order in calling the functions of the sensor, which is why I had to adapt it.

Reply:
Should be `time` and not `int` right?
Would be good to actually type and do it also in the library properly